admin/securedrop_admin/__init__.py

```diff
@@ -62,7 +62,10 @@
 from typing import Type
 
 sdlog = logging.getLogger(__name__)
-RELEASE_KEY = '22245C81E3BAEB4138B36061310F561200F4AD77'
+RELEASE_KEYS = [
+    '22245C81E3BAEB4138B36061310F561200F4AD77',
+    '324C978C1CD14C0C2929D7D96FE1D5E9814BE242',
+]
```

Comment:
If exactly two keys are supported (see later signature check), it might be good to add a comment here to explicitly note that limitation.

Reply:
Input very much welcome on GUI updater changes! And good point, there's no reason not to add comments immediately. Overall we'll still have a bootstrapping problem where key verification fails if a given Workstation wasn't booted up to run its updater for a long time (mostly likely for Admin Workstations, much less so for Journalist Workstations). But as long as we're clear in the documentation, I believe we're covered there.